### ISSUE TITLE
vertical_parameter_alignment rule generates incorrect warning for nested parentheses

### ISSUE THREAD

**Issue by garie** [realm/SwiftLint#1488]
Version 0.18.1

This code generates a warning:
```
    static func getAdditionalInfoTextField(data: (size: CGSize,
                                                  identifier: String,
                                                  placeHolder: NSAttributedString,
                                                  padding: CGRect,
                                                  borderColor: CGColor,
                                                  borderWidth: CGFloat,
                                                  required: Bool)) -> UITextField {
```

This code does not generate a warning:

```
    static func getAdditionalInfoTextField(data: (size: CGSize,
                                           identifier: String,
                                           placeHolder: NSAttributedString,
                                           padding: CGRect,
                                           borderColor: CGColor,
                                           borderWidth: CGFloat,
                                           required: Bool)) -> UITextField {
```

**Comment by marcelofabri** [comment]
Fixed by @jpsim in https://github.com/realm/SwiftLint/pull/1726 🎉
